src/daemon/daemon.cpp

```diff
@@ -362,6 +362,18 @@ std::string get_bridged_interface_name()
     return bridged_id.toStdString();
 }
 
+bool is_bridged_impl(const mp::VMSpecs& specs,
+                     const std::vector<mp::NetworkInterfaceInfo>& host_nets,
+                     const std::string& preferred_net)
```

Comment:
Same like the previous, maybe `preferred_net`->`prefered_net_name`.

Reply:
See reply above. I don't think this needs to be tied to the type. "name" is something that I would append if I also had a variable with the actual thing (e.g. file and file_name). In this context, I think it is pretty redundant and only makes the code longer and harder to read.

Reply:
Yep, it is really a minor thing. I think we can leave it like this.